modules/cnv_virtctl_commands.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
```

Comment:
Do you need this file? Looks identical to `modules/cnv-virtctl-commands.adoc` save for the underscores in the file name.

Reply:
No, in fact I thought I had already banished this to the dark clutches of the netherworld. Gone!